packages/destination-actions/src/destinations/fullstory/__tests__/var.test.ts

```diff
@@ -0,0 +1,106 @@
+import { normalizePropertyNames } from '../vars'
+
+const suffixToExampleValuesMap: Record<string, any[]> = {
+  str: ['some string'],
+  bool: [true, false],
+  // We include an int value with real example values since real will always be preferred when inferring
+  // custom var types from values.
+  real: [1, 1.23],
+  int: [1],
+  date: [new Date()],
+  obj: [{}, { secondLevelProp: 'some value' }]
+}
+
+suffixToExampleValuesMap.strs = [suffixToExampleValuesMap.str]
+suffixToExampleValuesMap.bools = [suffixToExampleValuesMap.bool]
```

Comment:
i might be misunderstanding the intention here, but is the goal to put these bools into a nested array? like so `[[true, false]]`

Reply:
That's the intent, correct. I wasn't 100% happy with how this shook out, but I was trying to reuse example values across single and array values for the various types. On line 37, you'll see that we iterate over each value, so `[[true, false]]` becomes `[true, false]` to test inferring the `_bools` type suffix -- whereas for `_bool` we create a distinct property for `true` and a distinct property for `false` to test that each of those values will result in a `_bool` type suffix.